### PR TITLE
NXP-25021: display a login banner when the session is expired

### PR DESCRIPTION
**Pull request by Gabez0r**
Depends on nuxeo/nuxeo-elements#27.

### CODE REVIEW

File: elements/nuxeo-app.html [hunk 1]
```diff
@@ -521,7 +524,8 @@
         'nuxeo-documents-deleted': '_documentsDeleted',
         'nuxeo-documents-untrashed': '_documentsUntrashed',
         'nuxeo-documents-dropped': '_documentsDropped',
-        'nuxeo-diff-documents': '_diffDocuments'
+        'nuxeo-diff-documents': '_diffDocuments',
+        'session-expired': '_handleExpiredSession'
```
Comment (afjusto) [inline elements/nuxeo-app.html]: would it work if we register the listener in the `nuxeo-expired-session` element instead?

**Comment by afjusto** [comment]
👍

File: elements/nuxeo-app.html (hunk 1 above)
Comment (Gabez0r) [reply elements/nuxeo-app.html]: It must be registered on a common parent, either on app or at the document level. :(

**Comment by guirenard** [comment]
:+1:

**Pull request merged**